net-misc/sipp/sipp-3.6.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="SIPp is a free Open Source test tool / traffic generator for the SIP protocol"
+HOMEPAGE="http://sipp.sourceforge.net/ https://github.com/SIPp/sipp/releases"
+SRC_URI="https://github.com/SIPp/sipp/releases/download/v${PV}/${PF}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Please verify this. I see multiple mentions about other licenses in LICENSE.txt, which also names few source files with different licenses.

Reply:
First line of LICENSE.txt:

SIPp as a whole is licensed under the GNU General Public License, version 2 or any later version (copy below).

But you are right, then there is this:

send_packets.c, send_packets.h, md5.c and md5.h have separate copyright
licenses, which apply only to those files.

The following license applies to send_packets.c and send_packets.h only:

For send_packets.c and send_packets.h ... not sure what to call that, guessing "as is" but I'm not seeing such a license in /usr/portage/licenses, and the aladdin license in tree doesn't math the aladdin supplied license in the LICENSE.txt.

I'm assuming that since those licenses don't really mind what they're going into we can just stick with GPL-2, alternatively I need to add LICENSES.txt into licenses as SIPP, and add it to the free group?

Reply:
Put ISC and GPL-2+ in the LICENSE field at least. Should be good enough, at least for me.